src/model/schema.js

```diff
@@ -667,6 +674,38 @@ export default class Schema {
 		return validRanges;
 	}
 
+	/**
+	 * Tries to find position ancestors that allows to insert given node.
+	 * It starts searching from the given position and goes node by node to the top of the model tree
+	 * as long as {@link module:engine/model/schema~Schema#isLimit limit element} or top-most ancestor won't be reached.
+	 *
+	 * @params {module:engine/model/node~Node} node Node for which allowed parent should be found.
+	 * @params {module:engine/model/position~Position} position Position from searching will start.
+	 * @params {module:engine/model/element~Element} [limitElement] Custom limit element.
+	 * @returns {module:engine/model/element~Element|null} element Allowed parent or null if nothing was found.
+	 */
+	findAllowedParent( node, position, limitElement ) {
```

Comment:
The limit element here is strange since the method also looks for an element with `isLimit` property. This is a limit element which is not a limit. Can we try to use a custom element as a base of the conversion which allows everything to be a child of it?

Reply:
We can drop this limit element and do it in a different way but still check schema according to real context instead of element that allows everything.

Reply:
Ok, lets do it in `_splitToAllowedParent`.